src/cli.rs

```diff
@@ -0,0 +1,49 @@
+use crate::data;
+use clap::{Parser, Subcommand};
+use std::path::PathBuf;
+use std::{env, io};
+
+#[derive(Debug, Parser)]
+#[clap(name = "rustig")]
+#[clap(about = "A git implemented in RUST", long_about = None)]
+struct Cli {
+    #[clap(subcommand)]
+    command: Commands,
+}
+
+#[derive(Debug, Subcommand)]
+enum Commands {
+    #[clap()]
```

Comment:
Question: quel est l'utilité de cette ligne ?

```suggestion
    /// Create an empty Git repository or reinitialize an existing one
```

Reply:
bien vu, je pense qu'on peut l'enlever